<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Paint</class>
 <widget class="QWidget" name="Paint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paint</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Options</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="rb_green">
          <property name="text">
           <string>Green pen</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_red">
          <property name="text">
           <string>Red pen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_blue">
          <property name="text">
           <string>Blue pen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_black">
          <property name="text">
           <string>Black pen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_orange">
          <property name="text">
           <string>Orange pen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_stiker">
          <property name="text">
           <string>Stiker</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_addNewStiker">
       <property name="text">
        <string>Add new stiker</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_clear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
